relayer/core/relay.go

```diff
@@ -218,11 +251,20 @@ func LoadConfig() (*Config, error) {
 	}
 	config.Eth.PrivateKey = strings.TrimPrefix(value, "0x")
 
+	// TODO: auto populate contract addresses
```

Comment:
do in this PR plz as it is related to general plumbing/structure of relayer

Reply:
https://github.com/Snowfork/polkadot-ethereum/commit/ec27500ba0c88d0268ac6faaa85f59f2ab3dbffd